NEWS.md

```diff
@@ -0,0 +1,55 @@
+
+# MMAction2 Release Notes
+
+## v0.6.0 (Master)
+
+### Models
+* Add NonLocal module in TSM and I3D (https://github.com/open-mmlab/mmaction2/pull/41)
+* Add SSN pipeline (https://github.com/open-mmlab/mmaction2/pull/33)
+* Add SSN losses (https://github.com/open-mmlab/mmaction2/pull/52)
+* Add CSN model (https://github.com/open-mmlab/mmaction2/pull/87)
+
+### Datasets
+
+* Support hmdb51 dataset preparation (https://github.com/open-mmlab/mmaction2/pull/60)
+
+### New Features
+
+* Support rawframe inference in demo (https://github.com/open-mmlab/mmaction2/pull/59)
+* Support output video or gif in demo (https://github.com/open-mmlab/mmaction2/pull/72)
+* Add script to encode videos (https://github.com/open-mmlab/mmaction2/pull/84)
+* Support `with_offset` for rawframe dataset (https://github.com/open-mmlab/mmaction2/pull/48)
+
+### Modelzoo
+
+* Update Slowfast (https://github.com/open-mmlab/mmaction2/pull/51)
+* Update TSN, TSM video checkpoints (https://github.com/open-mmlab/mmaction2/pull/50)
+* Add data benchmark for TSN (https://github.com/open-mmlab/mmaction2/pull/57)
+* Add slowonly data benchmark (https://github.com/open-mmlab/mmaction2/pull/77)
+
+### Bug Fixes & Improvements
+
+* `CosineAnnealing` Typo (https://github.com/open-mmlab/mmaction2/pull/47)
+* Bug fix in analyze_log (https://github.com/open-mmlab/mmaction2/pull/54)
+* Fix configs for localization test (https://github.com/open-mmlab/mmaction2/pull/67)
+* Fix the bug of generating HMDB51 class index file (https://github.com/open-mmlab/mmaction2/pull/69)
+* Fix demo (https://github.com/open-mmlab/mmaction2/pull/88)
+* Correct the sthv1/sthv2 rawframes filelist generate command (https://github.com/open-mmlab/mmaction2/pull/71)
+* Polish data preparation codes (https://github.com/open-mmlab/mmaction2/pull/70)
+* Add `target_resolution` in demo and fix some args (https://github.com/open-mmlab/mmaction2/pull/80)
+* Improve data preparation code (https://github.com/open-mmlab/mmaction2/pull/58)
+* Improve unittest coverage and minor fix (https://github.com/open-mmlab/mmaction2/pull/62)
+
+### Docs
+
+* Fix bsn configs filename in README.md (https://github.com/open-mmlab/mmaction2/pull/74)
+* Update docs for demo (https://github.com/open-mmlab/mmaction2/pull/81)
+
+### Notable Changes
```

Comment:
The categories are too many. I suggest using the following.

```
### Highlights

### Breaking Changes

### New features

### Bug fixes & Improvements
```

Reply:
This can be done before release. This NEWS is for logging during developing, which provides preliminary info for writing a good release note

Reply:
E.g., contributor may not know whether to put its pr into Highlight section.

Reply:
We may add the highlights section just before the release. Other parts can be organized as proposed so that we can spare the time in reorganizing it.